webapi/src/main/java/org/oruko/dictionary/web/rest/NameApi.java

```diff
@@ -34,13 +34,7 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Everything looks fine in this pull request and I can't wait to merge it :)
But before I do, I would like to make a little cosmetic request...Can you revert the wildcard import in favor of explicitly stating the imports? 

As you will notice, we are not using wildcard imports (except in the tests) in the codebase. This is just to keep things consistent.

Thanks!

Reply:
Ops, my fault. i Will make a second PR using explicitly imports.